src/main/java/com/thealgorithms/backtracking/ArrayCombination.java

```diff
@@ -18,16 +19,16 @@ private ArrayCombination() {
      * @return A list of all combinations of length k.
      */
     public static List<List<Integer>> combination(int n, int k) {
-        if (n < 0 || k < 0 || k > n) {
+        if (k < 0 || k > n) {
             throw new IllegalArgumentException("Wrong input.");
         }
 
         List<List<Integer>> combinations = new ArrayList<>();
-        combine(combinations, new ArrayList<>(), 0, n, k);
+        combine(combinations, new LinkedList<>(), 0, n, k);
         return combinations;
     }
 
-    private static void combine(List<List<Integer>> combinations, List<Integer> current, int start, int n, int k) {
+    private static void combine(List<List<Integer>> combinations, LinkedList<Integer> current, int start, int n, int k) {
```

Comment:
Using LinkedList instead of List in method parameters is considered bad practice as it provides tight coupling, and issues with extensibility.

Reply:
List doesn't seem to have a removeLast method

Reply:
It has. In code below, you can see that it already used. And code compile and pass tests.